tripleo-ciscoaci/deployment/opflex/opflex-agent-container-puppet.yaml

```diff
@@ -148,6 +148,50 @@ parameters:
        The starting backoff value for retrying of policy requests, in seconds. The
        backoff is exponential, starting at this value, and continuing up to 16 retries.
     type: number
+  OpflexStatisticsMode:
+    default: 'real'
+    type: string
```

Comment:
Since this is a string value, we should add a description that lists the supported values ("off" and "real").